stable_baselines3/common/base_class.py

```diff
@@ -545,15 +545,18 @@ def predict(
         state: Optional[np.ndarray] = None,
         mask: Optional[np.ndarray] = None,
         deterministic: bool = False,
-    ) -> Tuple[np.ndarray, Optional[np.ndarray]]:
+    ) -> Tuple[np.ndarray, Optional[Tuple[np.ndarray, ...]]]:
         """
-        Get the model's action(s) from an observation
+        Get the policy action and state from an observation (and optional state).
+        Includes sugar-coating to handle different observations (e.g. normalizing images).
 
         :param observation: the input observation
-        :param state: The last states (can be None, used in recurrent policies)
+        :param state: The last hidden states (can be None, used in recurrent policies)
         :param mask: The last masks (can be None, used in recurrent policies)
+            this correspond to beginning of episodes,
```

Comment:
Just to double confirm: these are supposed to be `True` on the first step of the episodes (not on the final step)?

Reply:
Another sidecomment, but I wonder if we should consider changing the name `mask`. I realize it is somewhat established as a way to tell where one sequences end, but I wonder if `episode_starts` or `dones` would be more clear. I do not see other situations where you would want to reset the hidden states.